go/go.mod

```diff
@@ -3,14 +3,14 @@ module github.com/valkey-io/valkey-glide/go/glide
 go 1.20
 
 require (
+	github.com/google/uuid v1.6.0
```

Comment:
why this change?

Reply:
This was a holdover from a rebase. It could be removed, but will still be in the final go.mod as it is used in many different tests for other commands.

Reply:
This was from my zadd PR and it was changed automatically when I ran a `go mod tidy` citing that uuid should be direct, should be part of main now